content/docs/command-reference/pull.md

```diff
@@ -74,6 +74,9 @@ Note that the command `dvc status -c` can list files referenced in current
 stages (in `dvc.yaml`) or `.dvc` files, but missing from the cache. It can be
 used to see what files `dvc pull` would download.
 
+> Note that in the case of `dvc import` data, `dvc pull` downloads from the
```

Comment:
same here as in `dvc push` - not related, probably not the best place

Reply:
Rolled back too.